pilot/pkg/bootstrap/server.go

```diff
@@ -1192,3 +1197,10 @@ func (s *Server) initWorkloadTrustBundle(args *PilotArgs) error {
 	log.Infof("done initializing workload trustBundle")
 	return nil
 }
+
+// DisableCAServer returns whether CA functionality is disabled in istiod.
+// It return true only if istiod certs is signed by Kubernetes and
+// workload certs are signed by external CA
+func (s *Server) DisableCAServer() bool {
```

Comment:
nit: rename as checkDisableCa()? or isDisableCa(). Not sure if this function needs to be exported outside the bootstrap package.

Reply:
changed to `isDisableCa()`